opentreemap/importer/models/species.py

```diff
@@ -263,6 +263,7 @@ def validate_itree_code_field(self):
         else:
             self.cleaned[fields.species.ITREE_PAIRS] = pairs
 
+    @transaction.atomic
```

Comment:
Right, a failure doesn't really matter -- there's no integrity problem if we correctly save the heartbeat and then fail to save the row or if we correctly save the row and then fail to save the heartbeat.

So I think we don't need the transactions. As to the speed question, my point was that those methods do a lot of time-consuming things before getting to the saves, which were the point of adding the transaction.